fundingmanager.go

```diff
@@ -152,6 +152,10 @@ type fundingConfig struct {
 	// cannot be located, then an error is returned.
 	SignMessage func(pubKey *btcec.PublicKey, msg []byte) (*btcec.Signature, error)
 
+	// SignNodeAnnouncement is used by the fundingManager to sign the
+	// updated self node announcements sent after each channel announcement.
+	SignNodeAnnouncement func(nodeAnn *lnwire.NodeAnnouncement) (*btcec.Signature, error)
```

Comment:
Rather than requesting a signature for a node announcement, we should instead simply be fetching the _latest_ pre-signed `NodeAnnouncment`. We currently create, and sign one at start up within the `Start` method of the `server` struct. 

If we take this path, then we avoid unnecessarily updating our the timestamp on our node announcement. By avoiding this update, we save the network a bit of bandwidth as nodes now don't need to add this (unchanged) announcement to their re-broadcast buffers.